src/string/mod.rs

```diff
@@ -29,6 +30,13 @@ mod std_impl;
 #[derive(Clone, Default, PartialEq, PartialOrd, Eq, Ord)]
 pub struct StringDeku(pub(crate) String);
 
+#[cfg(feature = "defmt")]
+impl defmt::Format for StringDeku {
```

Comment:
[ ] Create simple test
[ ] Move to a separate module to make it visible for future lib extensions